apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -200,3 +200,42 @@ function wpcom_track_global_styles( $blog_id, $post, $updated ) {
 	}
 }
 add_action( 'save_post_wp_global_styles', 'wpcom_track_global_styles', 10, 3 );
+
+/**
+ * Adds the global style notice banner to the custom launch bar controls.
+ *
+ * @param array $custom_controls List of custom controls.
+ *
+ * return array The collection of launch bar custom controls to render.
+ */
+function wpcom_display_global_styles_banner( $custom_controls ) {
+	// Do not show the banner if the user can use global styles.
+	if ( ! wpcom_should_limit_global_styles() ) {
+		return;
+	}
+
+	if ( method_exists( '\WPCOM_Masterbar', 'get_calypso_site_slug' ) ) {
+		$site_slug = WPCOM_Masterbar::get_calypso_site_slug( get_current_blog_id() );
+	} else {
+		$home_url  = home_url( '/' );
+		$site_slug = wp_parse_url( $home_url, PHP_URL_HOST );
+	}
+
+	$upgrade_url = 'https://wordpress.com/plans/' . $site_slug;
+
+	$title = __( 'Styles hidden', 'full-site-editing' );
```

Comment:
The domain got filled in automagically, I guess the domain name was chosen based on that it's the one that is used on this file.